android/settings.gradle

```diff
@@ -1,8 +1,8 @@
 rootProject.name = 'Integreat'
+include ':react-native-screens'
+project(':react-native-screens').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-screens/android')
 include ':react-native-gesture-handler'
 project(':react-native-gesture-handler').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-gesture-handler/android')
-include ':react-native-view-pdf'
```

Comment:
This change is also odd. why is the pdf view removed?

Reply:
It was in there twice, so I removed one occurence.